.env.example

```
@@ -1,65 +1,56 @@
-# To find your firm_id after signing in:
-#   Supabase Dashboard → Table Editor → firm_memberships → copy your firm_id
-# IMPORTANT: Each account must use its own unique firm_id.
-# Do NOT share or copy firm_ids between accounts — this would break
-# attorney-client privilege isolation.
-VITE_FIRM_ID=
-
-# ── Supabase (required for auth & cloud sync) ──
-# Create a project at https://supabase.com/
+# ── Public browser configuration ──
 VITE_SUPABASE_URL=
 VITE_SUPABASE_ANON_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_SUPABASE_ANON_KEY key should go before the VITE_SUPABASE_URL key

(UnorderedKey)